src/backend/utilities/configuration.ts

```diff
@@ -34,9 +35,9 @@ if (!baseUri) {
   throw new ConfigurationError('No base URI provided');
 }
 
-const did = env.DID || 'pending';
+const did = env.DID as DidUri;
```

Comment:
I think the `pending` part was important [here](https://github.com/BTE-Trusted-Entity/socialkyc.io/blob/main/src/backend/utilities/fullDid.ts#L104)

Reply:
I guess let’s go with that for now, to speed up the process. I am not very happy with the solution though. We’re telling the TS and the developers that configuration.did is always defined. At the same time we have a clear use case when it’s undefined. Feels like laying a trap for a future developer.